server/test/unit/documents.specs.js

```diff
@@ -9,26 +9,27 @@ describe('Documents Model', () => {
   describe('Create Document', () => {
```

Comment:
What test is this for? Models... I believe you can have them in a folder that depicts that **server/test/unit/models/documents.specs.js** or rename the files to communicate that _documentsModel.specs.js_. Thanks